integreat_cms/cms/migrations/0095_chat_beta_testing.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 4.2.13 on 2024-06-17 08:30
+
+import django.core.validators
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    Add fields fpr chat beta testing functionality
```

Comment:
```suggestion
    Add fields for chat beta testing functionality
```
Is it a typo or is it an abbreviation I don't know about? :thinking:

Reply:
Haha, no, that's a typo alright :D